examples/basic_mysql_connection/index.js

```diff
@@ -13,8 +13,23 @@ var connection = mysql.createConnection({
 
 connection.connect();
 
-connection.query('SELECT * FROM todos', function (error, results, fields) {
-    console.log(results);
-});
+var userInput = 'do your task';
+var sql = [
+		'INSERT INTO todos (name,due) VALUES(' + mysql.escape(userInput) + ', "2017-8-8")',
```

Comment:
Thank you for doing this in a method that involves user input, and thank you for using `mysql.escape()`! So far, you're the only person who did that.